tests/integration/integration-test.py

```diff
@@ -34,73 +35,308 @@ def setup(cls, config_path, openapi_path):
+    def test_get_games(self):
+        path = '/games'
+        resource = 'GameResource'
+        nullable_fields = ['score']
+        current_test_case = 'game_names'
+        for game_name in self.test_cases[current_test_case]:
+            with self.subTest('Test name query parameter',
+                              game_name=game_name):
+                params = {'name': game_name}
+                response = utils.test_endpoint(self,
+                                               path,
+                                               resource,
+                                               200,
+                                               query_params=params,
+                                               nullable_fields=nullable_fields)
+                response_data = response.json()['data']
+                self.assert_data_returned(current_test_case, response_data)
+                for row in response_data:
+                    returned_name = row['attributes']['name']
+                    self.assertEqual(game_name, returned_name)
+
+        current_test_case = 'game_developer_ids'
+        for developer_id in self.test_cases[current_test_case]:
+            with self.subTest('Test developerId query parameter',
+                              developer_id=developer_id):
+                params = {'developerId': developer_id}
+                response = utils.test_endpoint(self,
+                                               path,
+                                               resource,
+                                               200,
+                                               query_params=params,
+                                               nullable_fields=nullable_fields)
+                response_data = response.json()['data']
+                self.assert_data_returned(current_test_case, response_data)
+                for row in response_data:
+                    returned_id = row['attributes']['developerId']
+                    self.assertEqual(developer_id, returned_id)
+
+        current_test_case = 'scores'
+        for score_min in self.test_cases[current_test_case]:
+            with self.subTest('Test scoreMin query parameter',
+                              score_min=score_min):
+                params = {'scoreMin': score_min}
+                response = utils.test_endpoint(self,
+                                               path,
+                                               resource,
+                                               200,
+                                               query_params=params,
+                                               nullable_fields=nullable_fields)
+                response_data = response.json()['data']
+                self.assert_data_returned(current_test_case, response_data)
+                for row in response_data:
+                    returned_score = row['attributes']['score']
+                    self.assertLessEqual(int(score_min), returned_score)
```

Comment:
As mentioned above, the score as parameter should be a number and you shouldn't need to convert the type here.